### PR TITLE
topology2: Use input/output pin consistently

### PR DESCRIPTION
**Pull request by ranj063**
Use the input/putput pin terminology consistently to align with the kernel driver and avoid any confusion.

Signed-off-by: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>

### CODE REVIEW

File: tools/topology/topology2/cavs-nocodec.conf [hunk 1]
```diff
@@ -36,7 +36,6 @@
 <dai.conf>
```
Comment (kv2019i) [inline tools/topology/topology2/cavs-nocodec.conf:36]: "putput pin" ? :)

**Review by kv2019i — APPROVED** [review]
Looks good. One typo in commit, but that's it.

I did a double-check of all usages of "pin" in FW code and basicly 99% of usage is with "input pin" and "output pin". There is one usage of of "sink->id" in copier code, but even there the "sink->id" refers to copier's output, so this is not referring to "sink_pin".

So this seems to be a welcome alignment of terms.

File: tools/topology/topology2/cavs-nocodec.conf (hunk 1 above)
Comment (ranj063) [reply tools/topology/topology2/cavs-nocodec.conf:36]: fixed

File: tools/topology/topology2/include/common/output_audio_format.conf [hunk 2]
```diff
@@ -1,7 +1,7 @@
-# Class definition for spirce pin audio format object
```
Comment (plbossart) [inline tools/topology/topology2/include/common/output_audio_format.conf:1]: I like the spirce pin. That sounds like a good concept that's not overloaded just yet.

**Review by plbossart — APPROVED** [review]

**Review by aiChaoSONG — APPROVED** [review]

**Pull request merged**